src/views/GroupTable/GroupTable.tsx

```diff
@@ -1,54 +1,52 @@
-import React, { useState } from 'react';
+import React, { FC } from 'react';
 import { makeStyles } from '@material-ui/styles';
-
 import { UsersToolbar, UsersTable } from './components';
 import { Theme, Typography } from '@material-ui/core';
-import { RootState } from '../../store';
-import { connect } from 'react-redux';
+import { useDispatch } from 'react-redux';
 import { fetchGroupTable } from '../../store/groups/actions';
 import { UserGroupState } from '../../store/user/types';
+import useSelector from '../../hooks/useSelector';
+import { userGroupsSelector } from '../../store/selectors/user';
+import { groupTableViewSelector } from '../../store/selectors/groups';
 
 const useStyles = makeStyles((theme: Theme) => ({
-  root: {
-    padding: theme.spacing(3)
-  },
-  content: {
-    marginTop: theme.spacing(2)
-  },
-  center: {
-    textAlign: 'center'
-  }
+	root: {
+		padding: theme.spacing(3)
+	},
+	content: {
+		marginTop: theme.spacing(2)
+	},
+	center: {
+		textAlign: 'center'
+	}
 }));
 
-const mapStateToProps = (state: RootState) => {
-  return {
-    userGroups: state.user.groups,
-    tableView: state.groups.tableView
-  }
-}
-
-const GroupTable = (props: any) => {
-  const classes = useStyles();
-  const { tableView, dispatch, userGroups } = props;
-  if (tableView.selectedGroupId == 0) {
-    if (userGroups.length > 0) {
-      dispatch(fetchGroupTable(userGroups[0].groupId))
-    }
-  }
-  const currentGroup = userGroups.find((x: UserGroupState) => x.groupId == tableView.selectedGroupId)
-  return (
-    <div className={classes.root}>
-      <Typography className={classes.center} variant="h2">{currentGroup ? currentGroup.displayName : null}</Typography>
-      <UsersToolbar />
-      <div className={classes.content}>
-        <UsersTable users={tableView.table} />
-      </div>
-    </div>
-  );
+const GroupTable: FC = () => {
+	const classes = useStyles();
+	const dispatch = useDispatch();
+	const userGroups = useSelector(userGroupsSelector);
+	const tableView = useSelector(groupTableViewSelector);
+	if (tableView.selectedGroupId === 0) {
+		if (userGroups.length > 0) {
+			dispatch(fetchGroupTable(userGroups[0].groupId));
+		}
+	}
+	const currentGroup = userGroups.find(
+		(x: UserGroupState) => x.groupId === tableView.selectedGroupId
+	);
+	return (
+		<div className={classes.root}>
+			<Typography className={classes.center} variant='h2'>
+				{currentGroup ? currentGroup.displayName : null}
+			</Typography>
+			<UsersToolbar />
+			<div className={classes.content}>
+				<UsersTable users={tableView.table} className='TODO' />
```

Comment:
lets not add TODOs in the code (we can use the project tab to add todos not in code)

Reply:
fixed ea83af8cb046cdd621e29a64b6e7a08b8bfa7610
is there any intention for overriding the css rules?